app/components/primer/open_project/collapsible.ts

```diff
@@ -4,10 +4,19 @@ import {attr, target, targets} from '@github/catalyst'
 export abstract class CollapsibleElement extends HTMLElement {
   @target arrowDown: Element
   @target arrowUp: Element
+  @target triggerElement: HTMLElement
   @targets collapsibleElements: HTMLElement[]
 
   @attr collapsed = false
 
+  connectedCallback() {
+    this.triggerElement.addEventListener('keydown', event => {
+      if (event.key === 'Enter' || event.key === ' ') {
```

Comment:
That looks weird.. I'd prefer the same syntax for both, so 

```suggestion
      if (event.code === 'Enter' || event.code === 'Space') {
```